File: programme/check-even-or-odd-string/check-even-or-odd-string.cs
// Write a programme to check even or odd string
using System;

namespace Codinasion.Programme
{
    class EvenOrOddString
    {
        static void Main(string[] args)
        {
            Console.Write("Input  : ");
            String input = Console.ReadLine();

            Console.Write("Output : ");
            int length = input.Length;
            if (length % 2 == 0)
            {
                Console.WriteLine("Even string");
            }
            else
            {
                Console.WriteLine("Odd string");
            }
        }
    }
}

File: programme/check-even-or-odd-string/checkEvenOrOddString.jl
# Write a programme to check even or odd string

function lettersCounter(input)

    if (length(input)) % 2 == 0
        println("Even String")
    else
        println("Odd String")
    end
end

input = "Hello"
println("Input  : ", input)
print("Output : ")
lettersCounter(input)
